content/master/concepts/claim-server-side-apply.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Claim Server-Side Apply
+state: alpha
+alphaVersion: "1.15"
+weight: 61
+---
+
+When you create a claim, Crossplane creates a corresponding composite resource
+(XR). Crossplane keeps the claim in sync with the composite resource. When you
+change the claim, Crossplane reflects those changes on the composite resource.
+
+Read the [claims documentation]({{<ref "./claims">}}) to learn more about claims
+and how they relate to composite resources.
+
+Crossplane can use server-side apply to keep the claim in sync with the
+composite resource.
+
+{{<hint "tip">}}
+Use the `--enable-claim-ssa` feature flag to enable using server-side apply.
+
+Read the
+[Install Crossplane documentation]({{<ref "../software/install#feature-flags">}})
+to learn about feature flags.
+{{</hint>}}
+
+Server-side apply is a Kubernetes feature. When Crossplane uses server-side
+apply, the Kubernetes API server helps sync claims with composite resources.
+This fixes several bugs. Read more about server-side apply in the
+[Kubernetes documentation](https://kubernetes.io/docs/reference/using-api/server-side-apply/).
+
+
+If you see unexpected behavior when Crossplane syncs a claim with a composite
+resource, test whether enabling server-side apply fixes the problem. Server-side
+apply fixes the following problems:
+
+- Fields reappearing after you delete them from a claim's spec.
+- Crossplane not deleting labels and annotations from the composite resource
+  when you delete them from the claim.
+
+{{<hint "important">}}
+When you enable server-side apply, Crossplane is more strict about how it syncs
+claims with XRs:
+
+- The claim's `metadata` syncs to the composite resource's `metadata`.
+- The claim's `spec` syncs to the composite resource's `spec`.
+- The composite resource's `status` syncs to the claim's `status`.
+
+When you enable server-side apply Crossplane doesn't sync the composite resource's `metadata`
+and `spec` back to the claim's `metadata` and `spec`. It also doesn't sync the
+claim's `status` to the composite resource's `status`.
+{{</hint>}}
+
+## Use claim server-side apply with composition functions
```

Comment:
IMHO, there is no direct connection between claim server-side apply and usage of composition functions, and hence a reader might be confused. Perhaps the following paragraph should be moved somewhere under `composition-function.md`?

Reply:
The point is:

* There are additional benefits to end-to-end SSA. From claim -> XR -> composed.
* You only get end-to-end SSA when you use this alpha feature AND functions.

Given you have to use an alpha feature, I think the docs fit better "aligned" with the alpha feature (claim SSA) than the functions docs.